deploy/rbd/kubernetes/csi-rbdplugin-provisioner.yaml

```diff
@@ -148,6 +148,33 @@ spec:
               mountPath: /etc/ceph-csi-encryption-kms-config/
             - name: keys-tmp-dir
               mountPath: /tmp/csi/keys
+          ports:
+            - name: liveness-port
+              containerPort: 9808
+              protocol: TCP
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: liveness-port
+            initialDelaySeconds: 10
+            timeoutSeconds: 3
+            periodSeconds: 2
+            failureThreshold: 5
+        - name: liveness-probe
+          image: quay.io/k8scsi/livenessprobe:canary
```

Comment:
@Madhu-1 If I remember it right, we wanted to leverage liveness within cephcsi (cephcsi --type liveness) with a different metricspath (may be /healthz) and a different port (may be 8080)

And the  liveness code is complete, this PR should be focusing on yaml changes only.

correct ?

Reply:
Yes correct and we need to expose /healthz endpoint

Reply:
Which has to be taken care in the examples/service-monitor.yaml ?

Reply:
ignore, I was looking at a wrong place

Reply:
Have opened a New PR, Please check: https://github.com/ceph/ceph-csi/pull/1665